scripts/apps/authoring/authoring/services/RenditionsService.js

```diff
@@ -61,7 +61,8 @@ export function RenditionsService(metadata, $q, api, superdesk, _) {
      *  @param {boolean} isAssociated to indicate if picture is isAssociated or not
      *  @return {promise} returns the modified picture item
      */
-    this.crop = function(picture, isNew = true, editable = true, isAssociated = false) {
+    this.crop = function(picture, isNew = true, editable = true, isAssociated = false,
```

Comment:
imo would be better to take some options object instead of all the boolean params so it's obvious what your're setting there, the usage won't be like:
```js
return renditions.crop(scope.item, false, true, false, 'crop', true)
```
but rather
```js
return renditions.crop(scope.item, {isNew: true, editable: false, isAssociated: false, defaultTab: 'crop', showMetadata: true});
```